Content.Shared/Light/EntitySystems/LightReplacerSystem.cs

```diff
@@ -116,12 +105,11 @@ private void HandleInteract(EntityUid uid, LightReplacerComponent component, Int
     ///     using light replacer. Light fixture should have <see cref="PoweredLightComponent"/>.
     /// </summary>
     /// <returns>True if successfully replaced light, false otherwise</returns>
-    public bool TryReplaceBulb(EntityUid replacerUid, EntityUid fixtureUid, EntityUid? userUid = null,
+    private bool TryReplaceBulb(EntityUid replacerUid, EntityUid fixtureUid, EntityUid? userUid = null,
```

Comment:
For these, I think leaving them as public is fine. It's more nuanced but there may be something in the future that might use this as part of a call to the system.